internal/loadbalancebenchmark/config.go

```diff
@@ -0,0 +1,120 @@
+package loadbalancingbenchmark
+
+import (
+	"fmt"
+	"time"
+)
+
+type ClientGroup struct {
+	Cnt    int
+	Rps    int
+	LType  ListType
+	LUType ListUpdaterType
+}
+
+type LatencyConfig struct {
+	P50  time.Duration
+	P90  time.Duration
+	P99  time.Duration
+	P100 time.Duration
+}
+
+type ServerGroup struct {
+	Cnt           int
+	Type          MachineType
+	LatencyConfig *LatencyConfig
+}
+
+type TestConfig struct {
+	ClientGroup []ClientGroup
+	ServerGroup []ServerGroup
+	Duration    time.Duration
+}
+
+// return a normal latency config that satisfy the given rps
+func RpsLatency(rps int) *LatencyConfig {
+	// measure in millisecond
+	normal := time.Second / time.Duration(rps)
+	return &LatencyConfig{
+		P50:  time.Duration(int(0.4 * float32(normal))),
+		P90:  time.Duration(int(0.7 * float32(normal))),
+		P99:  time.Duration(int(0.8 * float32(normal))),
+		P100: time.Duration(int(2.0 * float32(normal))),
+	}
+}
+
+func (config *TestConfig) GetClientTotalCount() int {
```

Comment:
maybe `ClientCount()` or `NumClients()`, similar to related server function below

Reply:
embedded to `context` object